scripts/generate-app-config/generateAppConfig.js

```diff
@@ -28,7 +28,9 @@ const addConfig = (config, fileName, buildPath, fs, imports) => {
 /**
 * Writes out a file consisting of the app config and imports with the given file name to the specified path.
 */
-const generateAppConfig = (siteConfig, production, verbose, locale) => {
+const generateAppConfig = ({
+  siteConfig, mode, prefix, apps = [], verbose = false, basename,
```

Comment:
What is basename use for here? I'm assuming not the build path

might be helpful to add inline doc on the params

Reply:
I'm assuming this is the redirect basepath? https://github.com/cerner/terra-dev-site/pull/192/files?file-filters%5B%5D=.html&file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&file-filters%5B%5D=.md&file-filters%5B%5D=.png#diff-b8c0917cd285e586f733dc6bb78d432cL2

Reply:
basename is react routers basename prop
https://reacttraining.com/react-router/web/api/BrowserRouter/basename-string